code/modules/power/supermatter/supermatter.dm

```diff
@@ -1155,7 +1155,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		matter_power += 70 * object_size
 
 	//Some poor sod got eaten, go ahead and irradiate people nearby.
-	radiation_pulse(src, max_range = 6, threshold = 0.3, chance = 30)
+	radiation_pulse(src, max_range = 2 * object_size, threshold = 0.6 / object_size, chance = 10 * object_size)
 	for(var/mob/living/near_mob in range(10))
```

Comment:
irradiation range checked here too (does this even work? dont recognize these messages)

Reply:
that's just to warn admins and player that an irradiation might occur, doesn't actually irradiate

Reply:
yeah, but if the size of the object is 6 or greater, the warning will actually have a shorter range than the pulse, no?

Reply:
fair, lemme change it to have the same `2 * object_size` as the max rad pulse